src/Compilers/CSharp/Portable/Binder/Semantics/Operators/UnaryOperatorEasyOut.cs

```diff
@@ -42,58 +42,45 @@ internal static class UnopEasyOut
 
 
             private static readonly UnaryOperatorKind[] s_increment =
-                //obj   str   bool  chr   i08   i16   i32   i64   u08   u16   u32   u64   r32   r64   dec  
-                { ERR,  ERR,  ERR,  CHR,  I08,  I16,  I32,  I64,  U08,  U16,  U32,  U64,  R32,  R64,  DEC,
-                /* lifted: */ ERR,  LCHR, LI08, LI16, LI32, LI64, LU08, LU16, LU32, LU64, LR32, LR64, LDEC };
+            //obj  str  bool chr   i08   i16   i32   i64   u08   u16   u32   u64   r32   r64   dec  
```

Comment:
I don't think these formatting changes are necessary.

Reply:
These changes are reverting the file back to match `features/compiler`. It was edited in an earlier PR and that is the undo.